assets/js/pages/Layout/Layout.jsx

```diff
@@ -175,8 +174,8 @@ function Layout() {
             <div className="relative flex flex-col justify-end h-full px-8 md:w-full">
               <div className="relative p-5 flex items-center w-full space-x-8 justify-end mr-20">
                 <ProfileMenu
-                  username={username}
-                  email={email}
+                  username={userProfile.username}
```

Comment:
If we destruct userProfile we `could` do something like this
```
      <ProfileMenu
        username={username}
        email={email}
        logout={logout}
      />
    </div>
```

what do you think ?